install.md

```diff
@@ -327,6 +329,8 @@ To configure the installation settings:
    - `API-CPU-REQUEST` is the desired CPU resource request for the pods in the specified deployment. Default is 50m.
    - `API-MEMORY-REQUEST` is the desired memory resource request for the pods in the specified deployment. Default is 100Mi.
    - `API-REPLICA-COUNT` is the desired number of replicas for the specified deployment. Default is 1.
+   - `KUBERNETES-DISTRIBUTION` is the name of the kubernetes distribution. Defaults to "".
+       - Set "openshift" as the value when installing on an openshift environment.
```

Comment:
```suggestion
       - Set "openshift" as the value when installing on an OpenShift environment.
```

I think we should also be explicit that at this time you only need to set this when using OpenShift and that for all other distributions it should be left unset.

Reply:
I have added a line for the case of other distributions